code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -1243,14 +1240,18 @@ datum
 			on_mob_life(var/mob/M, var/mult = 1)
 				if (!M) M = holder.my_atom
 				if (!counter) counter = 1
-				if ((counter += (1*mult)) >= 10 && !fakedeathed)
+				switch(counter += (1 * mult))
+					if (1 to 9)
+						M.change_eye_blurry(10, 10)
+					if (10 to 18)
+						M.drowsyness  = max(M.drowsyness, 10)
```

Comment:
would prefer also not having the drowsy/blurry effect, but doesn't matter that much 🤷

Reply:
I think it's helpful in that (1) it lets you estimate when you're gonna "die" (2) it's useful for dramatics if you wanna scare people.